src/config.js

```diff
@@ -53,6 +54,7 @@ class Config {
     this.reauth = reauth || true
     this.retryDelay = retryDelay || 1000
     this.retryJitter = retryJitter || 500
+    this.retryMaxAttempts = retryMaxAttempts || 4
```

Comment:
`retryMaxAttempts` sounds like a number of attempts after first one failed. So if I wanted to have just one try and and no retries I would set this to zero (@mwan-ep  please confirm that this correct assumption). But if `retryMaxAttempts` is zero then `0 || 4` will set it to 4. Btw zero handling of zeros is the main difference between `||` operator and `??`, but we can't use `??` here. You need to manually check if `retryMaxAttempts` is set to undefined and use it then.

Reply:
Added a check for undefined and null here.